e2etest/bgptests/bgp.go

```diff
@@ -1416,3 +1478,17 @@ func substringCount(substr string) interface{} {
 		return strings.Count(action, substr)
 	}
 }
+
+// tryTranslate checks if given Community string is alias of community value
+// in Community CRs collection.
+func tryTranslate(community string, cs []metallbv1beta1.Community) string {
```

Comment:
As a general note (I suggested  to remove this), we could have a better name for this.
Why try? What happens if it fails?
communityForAlias would be a better name, and I'd return an error in case we don't find a valid alias.

Reply:
done.